js/stdlib/ts/ALGO.ts

```diff
@@ -1285,6 +1285,25 @@ const getDeletedApplicationInfoM = async (id: number): Promise<OrExn<AppInfo>> =
   }
 }
 
+let viewCache = new Map<string, Map<string, any>>();
```

Comment:
Use `const` for this because the container is constant but the contents are mutable